Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM fedora
+
+ENV TEST_USER_ID 1000
```

Comment:
Because I wanted to use variable `TEST_USER_ID` in below lines in `Dockerfile`.
How do you think?

```
RUN useradd -u $TEST_USER_ID tester
...
USER $TEST_USER_ID
```